tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{27,36}-dj111{,-grp}
-    py36-dj20{,-grp}
-    py36-dj21
+    py{27,36}-dj111{,-grp}-pil54
```

Comment:
If they all use the same pillow version, then it probably doesn't need to be included in the env name. (it can just go in the dependencies below without a specifier)